src/gtimelog/timelog.py

```diff
@@ -1080,6 +1080,8 @@ class TaskList(object):
 
     A TaskList has an attribute 'groups' which is a list of tuples
     (group_name, list_of_group_items).
+    It also has an attribute 'tasklist' which is an unsorted list of tasks,
+    potentially prefixed by 'group_name:'.
```

Comment:
There should be a blank line between paragraphs of text inside docstrings.

Also, I think instead of "unsorted" it should say "in source order".

(Any reason it couldn't be named `tasks`?  or `all_tasks`?)

Reply:
New, better suggestion: instead of keeping a list, build a dictionary from task name to its order in the original file.  Then reimplement `def index()` (renamed to `def order()`) to do a `return self.task_order.get(task, sys.maxsize)`.